operators/src/util/raster_stream_to_png.rs

```diff
@@ -23,6 +24,10 @@ pub async fn raster_stream_to_png_bytes<T, C: QueryContext>(
 where
     T: Pixel,
 {
+    // TODO: rather use `span.in_scope` because it is safer inside async functions (cf. https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.enter)
+    let span = span!(Level::TRACE, "raster_stream_to_png_bytes");
+    let _enter = span.enter();
```

Comment:
can we use in_scope here?

Reply:
yes but this would either mean wrapping the whole function body inside a closure or introduce another method, that's why I didn't do it yet.